babel.config.js

```diff
@@ -6,4 +6,9 @@ module.exports = {
     ['@babel/plugin-proposal-class-properties', { loose: true }],
     ['@babel/plugin-transform-parameters', { loose: true }],
   ],
+  env: {
+    test: {
+      presets: ['@babel/preset-typescript'],
```

Comment:
This was required to get `tests/utils/decorateWithThemeProvider.tsx` working. It's not really taking advantage of things right now but if we add more typed utilities it might.